dvpOAuth/__init__.py,sha256=V6CmNzD5psTjxhn8dz7o4olZlNUNT2GsISATppst8SM,163
dvpOAuth/class_OAuth.py,sha256=4xO-U2DMBKvRWlejuUT3i_s7ROdwFLisqyNV6673iAs,21725
dvpOAuth/error.py,sha256=Y3BjsevsSGMDVVdn6ztiQMcGNtB6lySkNPoL7y4BT-Q,131
dvpOAuth/util.py,sha256=A7ZoIm1mjYwD4X9aommSZRYpi9ZidUuqMd2OYf1Ee64,743
dvpOAuth-0.0.15.dist-info/LICENSE,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
dvpOAuth-0.0.15.dist-info/METADATA,sha256=PMh--edA1iQ7cLbwt-M2Uq4GnkvRDtlzP3IirPi6GD0,992
dvpOAuth-0.0.15.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
dvpOAuth-0.0.15.dist-info/top_level.txt,sha256=14UgmlstFHh0UvlWIljRlFO26u2OesXzFC8SsWFZRmw,9
dvpOAuth-0.0.15.dist-info/RECORD,,
